metadata.md

```diff
@@ -28,58 +38,87 @@ There may also be any number of delegated target roles.
+This file can optionally define other roles to which it delegates trust,
+or specifies that another role is trusted for some or all of the target files
+available from the repository. When delegated roles are specified, it is done
+so in a way similar to how the Root role specifies the top-level roles: by giving
+the trusted keys and signature threshold for each role. Additionally, one or more
+patterns are specified to indicate the target file paths for which clients
+should trust each delegated role.
 
 See [example](https://raw.githubusercontent.com/theupdateframework/tuf/develop/tests/repository_data/repository/metadata/targets.json) of Targets metadata.
 
 ## Delegated Targets Metadata (role1.json)
 
 Signed by: A delegated targets role.
 
-The metadata files provided by delegated targets roles follow exactly the same format as the metadata file provided by the top-level Targets role.
+A metadata file provided by a delegated targets role will follow exactly the same
+format as one provided by the top-level Targets role.
 
-When the targets role delegates trust to other roles, each delegated role provides one signed metadata file.  As is the
-case with the directory structure of top-level metadata, the delegated files are relative to the base URL of metadata available from a given repository mirror.
+When the targets role delegates trust to other roles, each delegated role will
+provide one signed metadata file.  As is the
+case with the directory structure of top-level metadata, the delegated files are
+relative to the base URL of metadata available from a given repository mirror.
 
 A delegated role file is located at:
 
 /DELEGATED_ROLE.json
 
-where DELEGATED_ROLE is the name of the delegated role that has been specified in targets.json.  If this role further delegates trust to a role named ANOTHER_ROLE, that role's signed metadata file is made available at:
+where DELEGATED_ROLE is the name of the role specified in targets.json.  If this
+role further delegates trust to a role named ANOTHER_ROLE, that role's signed
+metadata file would be found at:
 
 /ANOTHER_ROLE.json
 
 See
 [example](https://raw.githubusercontent.com/theupdateframework/tuf/develop/tests/repository_data/repository/metadata/role1.json)
 of delegated Targets metadata and [example](https://raw.githubusercontent.com/theupdateframework/tuf/develop/tests/repository_data/repository/metadata/role2.json) of a nested delegation.
 
-## snapshot Metadata (snapshot.json)
+## Snapshot Metadata (snapshot.json)
 
 Signed by: Snapshot role.
 
-The snapshot.json metadata file lists hashes and sizes of all metadata files other than timestamp.json. This file ensures that clients will see a consistent view of the files on the repository. That is, metadata files (and thus target file) that existed on the repository at different times cannot be combined and presented to clients by an attacker.
+The snapshot.json metadata file lists hashes and sizes of all metadata files
```

Comment:
Although once true, Snapshot now lists the version number of all metadata files.  See the Snapshot [example](https://raw.githubusercontent.com/theupdateframework/tuf/develop/tests/repository_data/repository/metadata/snapshot.json).

Reply:
fixed